src/edu/harvard/CS50.java

```diff
@@ -39,12 +39,41 @@
 
 package edu.harvard;
 
+import java.io.BufferedOutputStream;
+import java.io.FileDescriptor;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.Scanner;
 
 public class CS50
 {
+    // disable buffering on stderr and stdout
+    // only functional when class is initialized per http://docs.oracle.com/javase/specs/jls/se7/html/jls-12.html#jls-12.4.1
+    static
```

Comment:
Might any of these lines throw exceptions we should catch (and silently ignore)?